chef/cookbooks/provisioner/templates/default/crowbar_join.suse.sh.erb

```diff
@@ -456,6 +456,15 @@ if [ "$MODE" == "setup" -o "$MODE" == "start" ]; then
     # Get our hostname
     HOSTNAME=$(hostname -f)
 
+    # stop ntpd before we run ntpdate, and start it again afterwards.
+    ntp=ntp
+    service ntp status > /dev/null || ntp=ntpd
+    NTP_START=0
+    if service $ntp status > /dev/null; then
```

Comment:
Just extension of the previous comment to remove the $ntp variable which is the logic for setting the ntp/ntpd based of SLE version :).

But there could be an ideal "crowbar/chef" way of checking versions which may be given more preference.